Svc/SeqDispatcher/SeqDispatcherEvents.fppi

```diff
@@ -0,0 +1,31 @@
+event InvalidSequencer(
+    idx: U16
+) \
+    severity warning high \
+    format "Invalid sequence index {}"
+
+event NoAvailableSequencers() \
+    severity warning high \
+    format "No available cmd sequencers to dispatch a sequence to"
+
+event UnknownSequenceFinished(
+    idx: U16
+) \
+    severity warning low \
+    format "Sequencer {} said it finished a sequence but we didn't even know it had started one"
```

Comment:
It's a personal style thing, I don't shy away from being informal because it's direct and clear. That said, I think using pronouns here is a bit much, I can change it.